git-hub

```diff
@@ -1551,6 +1551,10 @@ class PullCmd (IssueCmd):
                 warnf('Checking out a closed pull request '
                     '(closed at {closed_at})!', **pull)
 
+            if pull['head']['repo'] is None:
+                die("It seems like the repository referenced by "
+                    "this pull request has been deleted")
```

Comment:
I would say "Can't checkout pull request {id}, no head repository found (maybe it was deleted?)", or something like that.